.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"]
+  }
+  
```

Comment:
Please format this file properly to get rid of the missing new line warning at the end of the file.